CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.33.0 Release
+- Release Date: July 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+ 
+### Breaking Changes
+ - Starting with mobile app v1.33.0, users on server versions below v5.19 may experience issues with how attachments, link previews, reactions and embed data are displayed. Updating your server to v5.19 or later is required.
+ 
+### Highlights
+
+### Improvements
+ - The server will now send a push notification to the client when the session has expired.
```

Comment:
not all servers.. not sure which version of the server does this.

Reply:
@srkgupta / @agnivade Would you know what cases https://mattermost.atlassian.net/browse/MM-25931 applies in?

Reply:
The server PR is labeled 5.26. So I would assume from 5.26 onwards. @wiggin77 can clarify further.

Reply:
Removed for now as this is upcoming improvement.